pom.xml

```diff
@@ -1200,6 +1210,11 @@
         <version>${spotless-maven-plugin.version}</version>
         <configuration>
           <java>
+            <includes>
+              <include>src/main/java/**/*.java</include>
+              <include>src/test/java/**/*.java</include>
+              <include>src/it/**/src/main/java/**/*.java</include>
+            </includes>
```

Comment:
I'll probably send an independent PR with these and the related formatting changes prior to merging this one.
Otherwise it will be hard to rebase other ongoing PRs (namely #5632)